From d39c4194e3a4fbb97491b48ea6feb5a6a4b7fd31 Mon Sep 17 00:00:00 2001
From: Michael Bestas <mkbestas@lineageos.org>
Date: Wed, 27 Feb 2019 00:17:33 +0200
Subject: [PATCH] Email: Fix generating id in android namespace

Fixes:
res/layout/preference_mailbox_item.xml:39: warn:
generated id 'android:id/title' for external package 'android'.

Change-Id: I93e613deae5b4b59d834a03f6c1ba828e361d889
---
 res/layout/preference_mailbox_item.xml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/res/layout/preference_mailbox_item.xml b/res/layout/preference_mailbox_item.xml
index a624d491a..f0271b5c1 100644
--- a/res/layout/preference_mailbox_item.xml
+++ b/res/layout/preference_mailbox_item.xml
@@ -36,7 +36,7 @@
         android:layout_marginEnd="8dip"
         android:layout_gravity="center" />
 
-    <TextView android:id="@+android:id/title"
+    <TextView android:id="@+id/title"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
         android:layout_weight="1"
-- 
2.17.1

